src/command/supergraph/resolve_config.rs

```diff
@@ -27,6 +27,24 @@ pub(crate) fn expand_supergraph_yaml(content: &str) -> RoverResult<SupergraphCon
         .and_then(|v| serde_yaml::from_value(v).map_err(RoverError::from))
 }
 
+#[cfg(test)]
+mod test_expand_supergraph_yaml {
```

Comment:
Do we want to add more tests here?

Reply:
yeah - i didn't add tests here at first because there were decent tests in `apollo-federation-types` - but we do enough funky stuff and had a regression here it's worth adding some more